data_collection_duration: 50 # seconds
num_of_iterations: 20 # number of iterations for training
num_of_epochs: 200 # number of epochs for training at each iteration
lr: 0.0005 # learning rate
weight_decay: 0.0 # weight decay